utils/selinux-python/Makefile

```diff
@@ -0,0 +1,155 @@
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=selinux-python
+PKG_VERSION:=3.1
+PKG_RELEASE:=1
+
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
+PKG_SOURCE_URL:=https://github.com/SELinuxProject/selinux/releases/download/20200710
+PKG_HASH:=f4d0a1a030bc291a6af498b26e0676b745075dd289a8ba16cdec86c3ea8f2f02
+PKG_BUILD_DEPENDS:=PACKAGE_selinux-audit2allow:libsepol
```

Comment:
missing `PKG_LICENSE`, `PKG_LICENSE_FILES` and `PKG_CPE_ID`

Reply:
Fixed.